### DISCUSSION TITLE
Where are the drafts saved?

### DISCUSSION THREAD

**Discussion by sallyFoster** [aeharding/voyager#1074] [Q&A] [answered]
I can never find the post drafts.

**Comment by aeharding** [comment]
Voyager doesn't support drafts atm.

**Reply by sallyFoster** [reply]
Oh, I see. I thought that it did because every time I leave a post unfinished and close the app, when I open the app, I get a pop up asking me if I want to save the draft.

**Reply by aeharding** [reply, marked as answer]
Oh, that is just recovered so any time you open the post text editor it will populate there.
